src/pipx/venv_inspect.py

```diff
@@ -172,7 +172,11 @@ def fetch_info_in_venv(venv_python_path: Path) -> Tuple[List[str], Dict[str, str
         impl_ver = sys.implementation.version
         implementation_version = "{0.major}.{0.minor}.{0.micro}".format(impl_ver)
         if impl_ver.releaselevel != "final":
-            implementation_version += impl_ver.releaselevel[0] + impl_ver.serial
+            implementation_version = "{}{}{}".format(
```

Comment:
Don't we need some periods `.` in this format string?

Reply:
This produces something like `3.10.0a1`, which is the preferred PEP 404 form. Dots can be added but will just be normalised away.

Reply:
Oh I see.  You nested `implementation_version` instead of doing the `+=`.  OK, I understand now.